[2025-05-18T02:50:30.576+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:50:27.736829+00:00/task_id=transform_api permission to 509
[2025-05-18T02:50:30.577+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:50:27.736829+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:50:27.736829+00:00/task_id=transform_api'
[2025-05-18T02:50:30.601+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:50:27.736829+00:00/task_id=transform_api permission to 509
[2025-05-18T02:50:30.602+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:50:27.736829+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:50:27.736829+00:00/task_id=transform_api'
[2025-05-18T02:50:30.637+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.transform_api manual__2025-05-18T02:50:27.736829+00:00 [queued]>
[2025-05-18T02:50:30.644+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.transform_api manual__2025-05-18T02:50:27.736829+00:00 [queued]>
[2025-05-18T02:50:30.644+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-05-18T02:50:30.658+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): transform_api> on 2025-05-18 02:50:27.736829+00:00
[2025-05-18T02:50:30.663+0000] {standard_task_runner.py:60} INFO - Started process 216 to run task
[2025-05-18T02:50:30.665+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'transform_api', 'manual__2025-05-18T02:50:27.736829+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmpwpqvk3vn']
[2025-05-18T02:50:30.667+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask transform_api
[2025-05-18T02:50:30.705+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:50:27.736829+00:00/task_id=transform_api permission to 509
[2025-05-18T02:50:30.706+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:50:27.736829+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:50:27.736829+00:00/task_id=transform_api'
[2025-05-18T02:50:30.707+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_dimensional_model.transform_api manual__2025-05-18T02:50:27.736829+00:00 [running]> on host 326f8e57e649
[2025-05-18T02:50:30.897+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='transform_api' AIRFLOW_CTX_EXECUTION_DATE='2025-05-18T02:50:27.736829+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-18T02:50:27.736829+00:00'
[2025-05-18T02:50:30.918+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-18T02:50:30.927+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=transform_api, execution_date=20250518T025027, start_date=20250518T025030, end_date=20250518T025030
[2025-05-18T02:50:30.960+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-18T02:50:30.980+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
